setup_lacework_agent.sh

```diff
@@ -79,6 +83,53 @@ render_agent_config() {
   echo "$_config_json" > "$LACEWORK_INSTALL_PATH/config/config.json"
 }
 
+add_default_syscall_config() {
+  local _syscall_config_yaml
+
+  # Default syscall config
+  _syscall_config_yaml="""
+etype.exec:
```

Comment:
Sorry for the late response, @rnalexlacework. We had a brief discussion on it today and we agreed on moving forward with disabling aggregation. I think it applies to all etypes, but want to confirm with @anilnanduri.

Reply:
Yes, please add 
`group-by:
 - none
 - `

to etype.exec, etype.initmod, etype.finimod

For etype.file please keep the aggregation as is, they are by file path and the process, which is needed to avoid lot of noise.